### PR TITLE
docs: add web monetization examples

### PR DESCRIPTION
**Pull request by sharafian**
Added some examples on how to do exclusive content with Web Monetization. Also included the examples in the static folder so users can try them out.

edit: also added a remove ads example
edit: added example counter and start/stop wm example

### CODE REVIEW

File: website/static/examples/show.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<head>
+  <meta name="monetization" content="$twitter.xrptipbot.com/Interledger">
```
Comment (traviscrist) [inline website/static/examples/show.html]: Should this use the coil payment pointer, not sure this one actually works...

Comment (sharafian) [reply website/static/examples/show.html]: https://twitter.com/Interledger it's certainly a legitimate twitter account. It feels weird to make the examples on webmonetization.org pay to Coil. If there were a verified charity I'd be good with using it in examples

Comment (sharafian) [reply website/static/examples/show.html]: I'll also include a note that this should be changed to your own payment pointer

Comment (traviscrist) [reply website/static/examples/show.html]: I agree that it ideally should not be coil, can we make both go to interledger then? I noticed the other one went to Coil.

**Pull request merged**